Mathlib/RingTheory/Coalgebra/Basic.lean

```diff
@@ -480,4 +480,25 @@ abbrev coalgebra [AddCommMonoid B] [Module R B] [Coalgebra R B] (e : A ≃ B) :
         ← LinearMap.comp_assoc, TensorProduct.map_map, ← TensorProduct.map_comp]
       simpa [LinearMap.comp_assoc, -coassoc_apply] using coassoc_apply (R := R) (A := B) _ }
 
+variable (R) in
+/-- Transfer `Coalgebra.IsCocomm` across an `Equiv`. -/
+lemma coalgebraIsCocomm [AddCommMonoid B] [Module R B] [Coalgebra R B] [IsCocomm R B] (e : A ≃ B) :
+    letI := e.addCommMonoid
+    letI := e.module R
+    letI := e.coalgebra R
+    IsCocomm R A :=
+  letI := e.addCommMonoid
+  letI := e.module R
+  letI := e.coalgebra R
+  { comm_comp_comul := by
+      -- TODO: `reassoc` for `LinearMap`
```

Comment:
Could you expand this with a few more words? Do you mean "implement `@[reassoc]`"? What would you tag with it if so?